lib/Makefile.am

```diff
@@ -19,6 +19,7 @@ include lib/scanner/list-scanner/Makefile.am
 include lib/scanner/kv-scanner/Makefile.am
 include lib/str-repr/Makefile.am
 include lib/secret-storage/Makefile.am
+include lib/logthrsource/Makefile.am
 
```

Comment:
I like that this goes to a separate directory. I was thinking about the same with the logthrdestination stuff, would make it easier to split a couple of files.